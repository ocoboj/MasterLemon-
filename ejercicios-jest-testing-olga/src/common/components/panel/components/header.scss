.header {
  background-color: #28A745;
  color: white;
}
