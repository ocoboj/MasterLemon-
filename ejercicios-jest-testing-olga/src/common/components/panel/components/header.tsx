import * as React from 'react';
const styles = require('./header.scss');

interface Props {
  title: string;
}

export const Header = (props: Props) => (
  <div className={`card-header ${styles.header}`}>
    <h3 className="panel-title">{props.title}</h3>
  </div>
);
