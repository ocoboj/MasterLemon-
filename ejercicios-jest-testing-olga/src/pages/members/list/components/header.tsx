import * as React from 'react';

export const Header: React.StatelessComponent = (props) => (
  <thead>
    <tr>
      <th>
        Avatar
      </th>
      <th>
        Id
      </th>
      <th>
        Name
      </th>
    </tr>
  </thead>
);
