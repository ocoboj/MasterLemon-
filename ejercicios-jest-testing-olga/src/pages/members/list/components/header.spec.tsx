import {Header} from './header';
import * as React from 'react';
import {shallow} from 'enzyme';

describe('pages/members/list/components/header specs', () => {
    it('should render as expected header', () => {
        const component = shallow(<header></header>);
        expect(component).toMatchSnapshot();
    });
});