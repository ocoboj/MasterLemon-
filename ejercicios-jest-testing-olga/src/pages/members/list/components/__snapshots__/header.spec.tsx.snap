// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`pages/members/list/components/header specs should render as expected header 1`] = `<header />`;
